src/common/composables/amcharts5/concepts-helper.ts

```diff
@@ -67,3 +68,27 @@ export const createBullet = (root: Root, settings: IBulletSettings): am5.Bullet
 export const createCircle = (root: Root, settings: ICircleSettings, circleTemplate: am5.Template<Circle>): am5.Circle => am5.Circle.new(root, {
     ...settings,
 }, circleTemplate);
+
+export const toggleSeries = (chart: am5.SerialChart, index: number) => {
+    if (chart instanceof am5percent.PieChart) {
+        const series = chart.series?.getIndex(0);
+        if (!series) return;
+
+        const slice = series.slices.values[index];
+        if (!slice || !slice.dataItem) return;
+
+        if (slice.isHiding() || slice.isHidden()) {
```

Comment:
Is `isHiding` and `isHidden` same thing ? 🤔

Reply:
[amcharts tutorial](https://www.amcharts.com/docs/v4/tutorials/toggling-series-via-api/#Toggling) recommends using both.

Reply:
I'll note it. Thanks a lot.